tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -2069,4 +2069,20 @@ class CompletionSuite extends BaseCompletionSuite {
     assertSingleItem = false
   )
 
+  check(
+    "overloaded-no-prefix",
+    """|trait Foo {
+       |  def overloaded(i: Int): Unit
+       |  def overloaded(s: String): Unit
+       |}
+       |trait Bar extends Foo {
+       |  val bar = overlo@@
+       |}
+       |""".stripMargin,
+    """|overloaded(i: Int): Unit
+       |overloaded(s: String): Unit
+       |""".stripMargin,
+    topLines = Some(2)
```

Comment:
Looks like the order in scala 3 is reversed:
```suggestion
    topLines = Some(2),
    compat = Map{
      "3" ->
         """|overloaded(s: String): Unit
            |overloaded(i: Int): Unit
            |""".stripMargin
    }
```